wiki/en/en-Software-Manual.md

```diff
@@ -261,4 +261,4 @@ your distance to the server is too large or your internet connection is not suff
 Overall Delay is calculated from the current Ping Time and the delay introduced by the current buffer settings.
 
 Audio Upstream Rate depends on the current audio packet size and compression setting. Make sure that the upstream
-rate is not higher than your available internet upload speed (check this with a service such as [speedtest.net](http://speedtest.net)).
+rate is not higher than your available internet upload speed (check this with a service such as [librespeed.org](https://librespeed.org/)).
```

Comment:
This is the only change which would have to go to changes.

Reply:
This reminds me - should we say anything about what a good jitter rate would be on the speed test? For example, on average less than 10ms or something? Might help people understand problems a bit better maybe.

Reply:
My speedtest.net does not show a jitter rate. Especially for jitter, it would be very path dependent.

Reply:
My speedtest.net does not show any jitter measurement. In any case, the jitter measurement would be path-dependent and the user would need to know how much of the path to the target test server is shared with the path to their Jamulus server.

Reply:
@trebmuh if you don't change the url but open a pr on changes for that, we could merge this into release.

Reply:
@gene96817 OK (although I was talking about https://librespeed.org) we can forget that then.

Reply:
Will open another PR against @release in a few minutes without the https://librespeed.org/ even if I do not understand why the other changes are fine but not the librespeed one.